src/main/groovy/com/ofg/uptodate/finder/FinderConfiguration.groovy

```diff
@@ -0,0 +1,11 @@
+package com.ofg.uptodate.finder
+
+import groovy.transform.PackageScope
+
+@PackageScope
+class FinderConfiguration {
```

Comment:
I've added an explicit constructor which doesn't allow me to use the @Immutable annotation. However, I've marked all the fields in the class final so the result should be the same.